plugin/kubernetes/metrics.go

```diff
@@ -0,0 +1,77 @@
+package kubernetes
+
+import (
+	"context"
+	"net/url"
+	"time"
+
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"k8s.io/client-go/tools/metrics"
+)
+
+var (
+	// requestLatency is a Prometheus Summary metric type partitioned by
```

Comment:
why are you calling it a Summary if it is a Histogram? https://prometheus.io/docs/practices/histograms/

Reply:
```suggestion
	// requestLatency measures latency of k8s client requests grouped by verb and url.
```